### PR TITLE
CLDR-15903 update charts

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-15903

- [ ] This PR completes the ticket.

### CODE REVIEW

**Comment by macchiati** [comment]
The charts software also generates coverageLevels.txt

File: common/properties/coverageLevels.txt [hunk 1]
```diff
@@ -95,6 +97,7 @@ pl ;	modern
 ps ;	modern
 pt ;	modern
 qu ;	moderate
+raj ;	basic
```
Comment (btangmu) [inline common/properties/coverageLevels.txt:100]: I wonder if this is related to https://unicode-org.atlassian.net/browse/CLDR-15954

**Review by btangmu — APPROVED** [review]

**Comment by srl295** [comment]
i should rerun my chart build after this i assume

**Comment by macchiati** [comment]
Yes, but let's wait until some of the other items come (like containment,...)

**Pull request merged**